src/views/pages/index.jsx

```diff
@@ -1,24 +1,21 @@
 import React from 'react';
-import { Link } from 'react-router';
 import { getAllModels } from '../../lib/api-client';
-import { getModelsForNewItem } from '../../lib/api-client';
+import { getAllItems } from '../../lib/api-client';
 export default class Index extends React.Component {
 
     allModels(e) {
         getAllModels();
     }
 
-    newItem(e) {
-      getModelsForNewItem();
+    allItems(){
+        getAllItems();
     }
-
     render() {
         return (
             <div id='index'>
                 <div id='links'>
                   <button onClick={this.allModels.bind(this)}>View all models</button><br/>
-                  <button onClick={this.newItem.bind(this)}>Create a New Item</button><br/>
-                  <Link to='/models/new'>Create a New Model</Link><br/>
+                  <button onClick={this.allItems.bind(this)}>View all Items</button><br/>
```

Comment:
`this.allItems.bind(this)` => `this.allItems`

The `.bind(this)` returns a new function which calls the original function while keeping the `this` context. If you don't need to use `this` in the original function, you don't need to include the `.bind(this)` here.